packages/components/src/table/summary.js

```diff
@@ -35,3 +35,13 @@ TableSummary.propTypes = {
 };
 
 export default TableSummary;
+
+export const TableSummaryPlaceholder = () => {
```

Comment:
Please write a snapshot test for this.

Reply:
Addressed https://github.com/woocommerce/woocommerce-admin/pull/7294/commits/d7e33983479498d945dc6af2b7de14abb16ad006

However, instead of writing snapshot tests for only this component `TableSummaryPlaceholder`, I've added snapshot tests for `TableCard`, which consumes this component.

IMHO, this makes more sense as `TableCard` is not having any snapshot test at the moment.

Reply:
@htdat fwiw we don't really do snapshot tests, they don't have high value.

Reply:
Thanks @samueljseay, I've removed snapshot tests in the last commit a2da976